Engine/Graphics/OpenGL/RenderOpenGL.cpp

```diff
@@ -2951,11 +2941,11 @@ void RenderOpenGL::RenderTerrainD3D() {
     GLint thistex = -1;
 
     // tile culling maths
-    int camx = WorldPosToGridCellX(pIndoorCameraD3D->vPartyPos.x);
-    int camy = WorldPosToGridCellY(pIndoorCameraD3D->vPartyPos.y);
-    int tilerange = (pIndoorCameraD3D->GetFarClip() / terrain_block_scale)+1;
+    int camx = WorldPosToGridCellX(pCamera3D->vCameraPos.x);
+    int camy = WorldPosToGridCellY(pCamera3D->vCameraPos.y);
+    int tilerange = (pCamera3D->GetFarClip() / terrain_block_scale)+3;
```

Comment:
Where is this +3 coming from?

Reply:
Arbitrary number to reduce the worst of terrain tile pop in.